scripts/deploy-operator-crd-scaling.sh

```diff
@@ -19,4 +19,12 @@ oc wait deployment new-pro-controller-manager -n "$TNF_EXAMPLE_CNF_NAMESPACE" --
 
 make addrole
 kubectl apply -f config/samples --validate=false
+for (( i=1; i<=15; i++ ))
```

Comment:
Why does the sleep is needed before the wait command? Maybe the wait command is not proper, it should be `--for condition=Available=True`, see:
 https://fabianlee.org/2022/01/27/kubernetes-using-kubectl-to-wait-for-condition-of-pods-deployments-services/

Reply:
I think this is here because the `jack` deployment actually does not exist for a little bit waiting on the operator to deploy it.  So the initial lookup fails immediately.

Reply:
Right, I think, I [saw it](https://github.com/test-network-function/cnf-certification-test-partner/blob/c73a30248e69a19d7e85f0f6ab26e37ebac22526/scripts/deploy-community-operator.sh#L73). `sleep 30` before the `wait` is enough with comment.

Reply:
@rdavid , the `sleep` was [her first implementation](https://github.com/test-network-function/cnf-certification-test-partner/pull/224/commits/bfe89cd25beb9f78cb0292eb22ad0b03dca9fee8), but [I suggested her ](https://github.com/test-network-function/cnf-certification-test-partner/pull/224#discussion_r1127605545)to change it for a check loop instead, in order to speed up the whole process a little bit: why waiting 30 secs if the resource is already available right after 4 or 12 secs?

Reply:
The error on exit on the loop timeout is not handled. I'd:
```
BIT=5
NUM=15
for i in $(seq 1 $NUM); do
  sleep $BIT
  kubectl get deployment jack -n "$TNF_EXAMPLE_CNF_NAMESPACE" || continue
  oc wait deployment jack -n "$TNF_EXAMPLE_CNF_NAMESPACE" --for=condition=available --timeout=240s
  exit
done
printf >&2 'Exit by timeout after %d seconds.\n $((BIT * NUM))
exit 1
```